modules/aws_iam_policy/aws-iam-policy.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "aws-iam-role"
+linkTitle: "aws-iam-role"
+date: 2021-06-06
+draft: false
+weight: 1
+description: Creates an IAM policy
+---
+
+This module can be used to create and manage an AWS IAM policy via opta
+
+### Example
+
+```
+  - name: deployer
+    type: aws-iam-role
```

Comment:
?
this is not an example of aws iam policy

Reply:
My bad, let me add a proper example for the same.